### DISCUSSION TITLE
OpenROAD installation on RHEL 7.9

### DISCUSSION THREAD

**Discussion by tfawal** [The-OpenROAD-Project/OpenROAD#5798] [General]
Hello everyone,

Not sure if this is the correct place to post this, but here goes.  I am trying to install OpenROAD on an RHEL 7.9 system via the local installation method.  When I run the setup.sh file in the OpenROAD-flow-scripts repo that I downloaded, it says the RHEL is not supported.

I found this closed thread on the same issue, but it did not give a clear answer as to whether this can work:

https://github.com/The-OpenROAD-Project/OpenROAD/issues/1207

I am posting this again to see if anyone can confirm this work around can be done and/or is safe.  This is for a project at work, and I don't want to mess up our system in any way, and I do not consider myself an expert with Linux OS, although I do have some experience.

My plan is to go through setup.sh and DependencyInstaller.sh, and cherry pick the CentOS bits to create my own setup.sh script.  Any input, ideas, or advice is appreciated.  Thank you.

**Comment by maliberty** [comment]
We do intend to support RHEL8 in the future.  I think your plan is reasonable.

**Reply by tfawal** [reply]
Ok great, thank you for the reply.  I will try the work around, and post the results here--successful or not.  If it does t work I'll share the script as well for others that might need it.

**Reply by tfawal** [reply]
Just wanted to follow up that this method was not finished to completion.  I did write a modified installation script cherry-picking the CentOS 7 bits, but there were still package compatibility issues along the way.  I got about 75% or so of the way through the installation, but decided it was not worth the time to continue at that point.

I wound up installing a Debian 12 VM on RHEL 7.9, and it installed just fine.  Thanks to maliberty for the response and help, I am closing this thread now.

**Comment by maliberty** [comment]
FYI we are working on adding RHEL9 support
